.github/pull_request_template.md

```diff
@@ -3,13 +3,13 @@ Please provide a description for what this PR is for.
 And to help us figure out who should review this PR, please 
 put an X in all the areas that this PR affects.
 
-[ ] Configuration Infrastructure
-[ ] Docs
-[ ] Installation
-[ ] Networking
-[ ] Performance and Scalability
-[ ] Policies and Telemetry
-[ ] Security
-[ ] Test and Release
-[ ] User Experience
-[ ] Developer Infrastructure
+- [ ] Configuration Infrastructure
```

Comment:
Would you remove this file from this PR as you have it in #10044. Thanks.

Reply:
Yep sorry, I messed it up in origin 🤦

Reply:
I will force push since there is a commit that should not be here.